walkthroughs/howto-tls-file-provided/infrastructure/ecs-service.yaml

```diff
@@ -400,8 +400,9 @@ Resources:
     Type: AWS::ElasticLoadBalancingV2::TargetGroup
     Properties:
       HealthCheckIntervalSeconds: 6
-      HealthCheckPath: /ping
+      HealthCheckPath: /ready
       HealthCheckProtocol: HTTP
```

Comment:
Why do we need a HTTP based health check? It would be simple to make this TCP based.

Reply:
We don't need it, but this indicates that the envoy is ready to serve traffic: https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--ready